src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -931,5 +932,107 @@ public void WorkingDirectory_Set_GetReturnsExpected(string workingDirectory)
             var info = new ProcessStartInfo { WorkingDirectory = workingDirectory };
             Assert.Equal(workingDirectory ?? string.Empty, info.WorkingDirectory);
         }
+
+        [Fact(Skip = "Manual test")]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)]
+        public void StartInfo_WebPage()
+        {
+            ProcessStartInfo info = new ProcessStartInfo
```

Comment:
I don't think we can kill the browser since that would be obnoxious if I ran the tests outer loop locally (however rarely I might do that). Popping and leaving a microsoft.com tab up, I think is reasonable. Since URLs are important to spuport it seems worth validating that the process start succeeds in that case.